dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -791,23 +772,6 @@ class ScriptsControllerTest < ActionController::TestCase
     assert unit.script_levels.any?
   end
 
-  test 'updates teacher resources' do
-    sign_in create(:levelbuilder)
-    Rails.application.config.stubs(:levelbuilder_mode).returns true
-
-    unit = create :script
-    stub_file_writes(unit.name)
-
-    post :update, params: {
-      id: unit.id,
-      script: {name: unit.name},
-      script_text: '',
-      resourceTypes: ['curriculum', 'vocabulary', ''],
-      resourceLinks: ['/link/to/curriculum', '/link/to/vocab', '']
-    }
-    assert_equal [['curriculum', '/link/to/curriculum'], ['vocabulary', '/link/to/vocab']], Script.find_by_name(unit.name).teacher_resources
-  end
```

Comment:
Legacy teacher resources can no longer be updated. Instead, they will be migrated to become migrated teacher resources, and then they will become editable again. for the rest of the plan there, see the [migrating teacher resources](https://docs.google.com/document/d/1yIL2Z7aqFTdbCi24-RpUhPO_B3wRdsiKRTB1EEs17EY/edit) doc.